content/en/docs/security/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Security"
+description: "Security Processes and Guidelines"
+weight: 68
+---
+
+Kyverno serves an admission controller and is a critical component of the Kubernetes control-plane. It is important to properly secure and monitor Kyverno. This section provides guidance on securing Kyverno and the security processes for the Kyverno project.
+
+## Disclosure Process
+
+Security vulnerabilities are best handled swiftly and discretely with the goal of minimizing the total time users remain vulnerable to exploits.
+
+If find or suspect a vulnerability please send an email to the security group at kyverno-security@googlegroups.com with the following information:
+- description of the problem
+- precise and detailed steps (include screenshots) that created the problem
+- the affected version(s)
+- any known mitigations
+
+Kyverno security response team will send an initial acknowledgement of the disclosure in 3-5 working days. Once the vulnerability and mitigation are confirmed, the team will plan to release any necessary changes based on the severity and complexity. Additional details on the security policy and processes are available in the Kyverno [git repo](https://github.com/kyverno/kyverno/blob/main/SECURITY.md).
+
+## Contact Us
+
+To communicate with the Kyverno team, for any questions or discussions, use [Slack](https://slack.k8s.io/#kyverno) or [GitHub](https://github.com/kyverno/kyverno).
+
+
+## Issues
+
+All security related issues are labeled as `security` and can be viewed with this query:
+
+  [https://github.com/kyverno/kyverno/labels/security](https://github.com/kyverno/kyverno/labels/security)
+
+
+## Release Artifacts
+
+With every release, Kyverno uploads the following artifacts:
```

Comment:
Although they aren't on the releases page, we also upload the container images stated here: https://github.com/orgs/kyverno/packages?repo_name=kyverno